documents/policies/external/Competition Requirements.md

```diff
@@ -94,96 +115,124 @@ The WCA Delegate must submit the following information when requesting approval
-         2. It must be stated what the deadline is to be accepted for a registrant on the waiting list.
-            1. This deadline should be the same as the deadline for refunds for competitors canceling their registration, if (partial) refunds are to be given.
-         3. The organization team may specify a limit to the waiting list. Once the limit of the waiting list has been reached, the Delegate should inform the WCAT to close the registration period. 
-            1. In case a limit was not set in advance, the Delegate may contact the WCAT to request to have the registration period closed if the waiting list has become unreasonably long.
-   6. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+      3. Exception: for competitions with only one event, the deadline for adding registered events may be omitted.
+      4. At the discretion of the WCA Delegate(s), a competitor who is deemed to have made a mistake with the user interface during the registration process (e.g. registering for the inverse list of events that they intended) can be granted an exception to the deadline for adding registered events.
+   5. The registration requirements may include the conditions of the waiting list if one is to be used. Competitions are expected to use a waiting list unless specified by the organization team.
+      1. Registrants will be placed on the waiting list if they have completed all requirements for a valid registration, but the competitor limit has been filled. Registrants from the waiting list will be accepted in the order that their registration was completed once a spot becomes available.
+      2. It must be stated what the deadline is for accepting a registrant on the waiting list.
+         1. The default deadline is the time at which registration closes.
+         2. This deadline should be the same as the deadline for refunds for competitors canceling their registration, if refunds are to be given.
+      3. The organization team may specify a limit to the waiting list. Once the limit of the waiting list has been reached, the Delegate should close the registration period. 
+   6. The registration requirements may include a policy on registration refunds if offered for the competition.
+      1. A full refund may deduct any necessary transaction fees from the registration fee.
+      2. There must be a deadline after which refunds are not issued.
+         1. This deadline should be the same as the deadline to add registered events except if the deadline to add registered events is during the competition.
+      3. Under force majeure, a full refund of their registration fee may be issued beyond the deadline at the delegate’s discretion.
```

Comment:
I don't think the WCA Delegate should be the one deciding on refund conditions. Refunds are a financial decision, and competition finances should be handled by organisers. I don't think organisers should have to ask for permission from a WCA representative to apply a force majeure refund policy.

Reply:
In principle I agree, but the fact that payments for most competitions are taken from within the WCA website itself (even if it's to a non-WCA Stripe account) requires the WCA to have some accountability in how refunds are processed.

If payments are taken entirely on a 3rd-party site I'd personally be fine with waiving any refund requirements.

Reply:
There are already many rules in thos policy for how refunds etc need to be handled (deadlines, waiting lists, etc) which organisers need to follow. I agree that given they are using the WCA website there needs to be some quality standards applied, but I don't think this is one of them.

More generally the WCA needs to have rules that acknowledge that Organisers do all the work, and WCA Delegates ensure rules are followed (an oversight role). I think that's mostly there, but a lot of the rules are written in a way where it assumes the organiser is a 14 year old who doesn't know what they're doing and the Delegate is implicitly expected to be the one doing the work and making the key decisions (e.g. deciding to issue refunds). Maybe that works for some regions of the US, but it doesn't work when the organiser is a legal entity with significant turnover and where, quite frankly, the people making financial decisions are more competent than the average WCA Delegate. Any rules that are set which restrict organisers apply to Speedcubing Australia Inc just as much as your random organiser - so they need to be set in such a way to be flexible.